init.lua

```diff
@@ -54,8 +63,5 @@ for file in lfs.dir(locale_dir) do
     end
 end 
 
--- Path of the configuration
-conf_path = "/etc/ssowat/conf.json"
-
 -- You should see that in your Nginx error logs by default
 ngx.log(ngx.INFO, "SSOwat ready")
```

Comment:
Why you don't use ``` local logger = require("log")``` here? (noob here)

Reply:
Not sure either, I tried at some point, but this init.lua is kinda special because that's where the conf_path gets defined, which is needed to load the `log` module ... and also there's the thing that if you want people to be able to tweak ssowat settings without having to restart/reload nginx, you need to load the settings outside of `init.lua` ... (Not 100% clear for me, sorry)